camayoc/tests/qpc/ui/conftest.py

```diff
@@ -6,6 +6,13 @@
 from camayoc.utils import get_qpc_url
 
 
+def pytest_configure(config):
+    """Check for presence of driver for selenium."""
+    if not config.option.driver:
+        setattr(config.option, 'markexpr', 'not driver')
```

Comment:
Can you expand the doc string to explain what happens. I'm not sure what is going on here.